lightning/src/routing/router.rs

```diff
@@ -536,13 +545,13 @@ pub fn find_route<L: Deref, S: Score>(
 where L::Target: Logger {
 	get_route(
 		our_node_pubkey, &params.payee, network, first_hops, params.final_value_msat,
-		params.final_cltv_expiry_delta, logger, scorer
+		params.final_cltv_expiry_delta, params.max_total_cltv_expiry_delta, logger, scorer
 	)
 }
 
 pub(crate) fn get_route<L: Deref, S: Score>(
 	our_node_pubkey: &PublicKey, payee: &Payee, network: &NetworkGraph,
-	first_hops: Option<&[&ChannelDetails]>, final_value_msat: u64, final_cltv_expiry_delta: u32,
+	first_hops: Option<&[&ChannelDetails]>, final_value_msat: u64, final_cltv_expiry_delta: u32, max_total_cltv_expiry_delta: u32,
```

Comment:
nit: move `max_total_cltv_expiry_delta` to following line to keep with our (admittedly loose) 100-char per line limit.